schemas/com.snowplowanalytics.iglu/igluctl_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,274 @@
+				"rawMode": {
+					"description": "Produce raw DDL with Snowplow-specific data or not",
+					"type": [
+						"boolean",
+						"null"
+					],
+					"default": false
+				},
+				"splitProduct": {
+					"description": "Split product types (like [string,integer]) into separate columns or not",
+					"type": [
+						"boolean",
+						"null"
+					],
+					"default": false
+				},
+				"noHeader": {
+					"description": "Place header comments into output DDL or not",
+					"type": [
+						"boolean",
+						"null"
+					],
+					"default": false
+				},
+				"force": {
+					"description": "Force overwrite existing manually-edited files",
+					"type": [
+						"boolean",
+						"null"
+					],
+					"default": false
+				}
+			},
+			"required": [
+				"output",
+				"dbschema",
+				"owner",
+				"db",
+				"varcharSize",
+				"withJsonPaths",
+				"rawMode",
+				"splitProduct",
+				"noHeader",
+				"force"
+			]
+		},
+		"actions": {
+			"description": "Configuration for s3cp and push commands",
+			"type": [
+				"array",
+				"null"
+			],
+			"items": {
+				"type": "object",
+				"anyOf": [
+					{
+						"description": "Configuration for static push command",
+						"type": "object",
+						"properties": {
+							"action": {
+								"type": "string",
+								"enum": [
+									"push"
+								]
+							},
+							"registry": {
+								"type": "string",
+								"format": "uri"
+							},
+							"isPublic": {
+								"type": "boolean",
+								"default": false
+							},
+							"apikey": {
+								"oneOf": [
+									{
+										"format": "uuid"
+									},
+									{
+										"pattern": "^\\$[A-Z0-9-_]+"
+									}
+								]
+							}
+						},
+						"required": [
+							"action",
+							"registry",
+							"isPublic",
+							"apikey"
+						]
+					},
+					{
+						"description": "Configuration for static s3cp command",
+						"type": "object",
+						"properties": {
+							"action": {
+								"type": "string",
+								"enum": [
+									"s3cp"
+								]
+							},
+							"uploadFormat": {
+								"type": "string",
+								"enum": [
+									"jsonpath",
+									"jsonschema"
+								]
+							},
+							"bucketPath": {
+								"type": "string",
+								"pattern": "^s3://"
+							},
+							"accessKeyId": {
+								"description": "AWS Access Key Id",
+								"type": "string",
+								"pattern": "^\\$[A-Z0-9-_]+"
+							},
+							"secretAccessKey": {
+								"description": "AWS Secret Access Key",
+								"type": "string",
+								"pattern": "^\\$[A-Z0-9-_]+"
+							},
+							"profile": {
+								"description": "AWS profile",
+								"type": [
+									"string",
+									"null"
+								]
+							},
+							"region": {
+								"description": "AWS region",
+								"type": [
+									"string",
+									"null"
+								],
+								"default": "us-west-2"
+							}
+						},
+						"required": [
+							"action",
+							"uploadFormat",
+							"bucketPath",
+							"accessKeyId",
```

Comment:
Most likely we'll never have keys in config. They shouldn't be required.

Reply:
AWS has multiple was to get permissions inside the code:

1. `profile`
2. `DefaultAWSCredentialsProviderChain`

And few more, but those two are already implented in igluctl.